tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1008,8 +1008,9 @@ def func(self, xp, a, b):
     'scalar_value': [-3, 3, -1.2, 1.2],
```

Comment:
Could you add edge cases for scalar/array?

Reply:
Fixed.